Studio-Lovelies/GG-JointJustice-Unity
# Narrative scripts finish before playing game over script

## Summary

Closes #317

## Testing instructions

Start the game and go to part 3 of Jory's Trial.
Present incorrect evidence until you have one life.
Present evidence again.
Notice how it doesn't fade to black until the failure dialogue has finished.
This is most noticeable on the script TMPHFAIL3 where the ISSUE_PENALTY command is used early.

## Additional information

- `[ ]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource:
